qmi/instruments/picotech/_picoscope.py

```diff
@@ -352,3 +354,7 @@ def get_block_data(self, channels: List[int]) -> Tuple[np.ndarray, float, List[b
     @rpc_method
     def run_block(self, num_pretrig_samples: int, num_posttrig_samples: int, timebase: int) -> None:
         raise NotImplementedError("Method 'run_block' is not implemented in the base class")
+
+    @rpc_method
+    def get_time_resolution(self, time_base: int) -> float:
```

Comment:
I would call this `get_sampling_interval` as that matches with the API docs

Reply:
YEs, that's a good one.